### PR TITLE
Locally test with azure-sdk baseurl

### PR DESCRIPTION
**Pull request by heaths**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ app_insights: 5e03f19a-f5b4-4620-97d5-35707cec7fcb
 # The preview server settings
 host: 127.0.0.1
 port: 4000
+baseurl: /azure-sdk
```
Comment (vhvb1989) [inline _config.yml:24]: Does it bring some benefit v/s localhost?  just curious...??  or is it just for visually looking this in the address bar?

Usually I like to see the `localhost` in the address bar when running locally to easily identity I am running locally.  And before pushing to master or creating PR, I push my fork/branch to the gh-pages branch to see the web live (as an staging env to demo my changes)

Comment (heaths) [reply _config.yml:24]: This just makes it so the base url e.g. azure.github.io/azure-sdk is the same in localhost. I made a recent change and forgot to pass a URL through `relative_url` so testing locally was successful, but it broke in production.

This isn't suppose to affect production, but isn't officially documented. One of the devs on GitHub Pages blogged about it, though: https://byparker.com/blog/2014/clearing-up-confusion-around-baseurl/

**Comment by weshaggard** [comment]
Looks like we might need to make some tweaks to the CI https://github.com/Azure/azure-sdk/blob/master/eng/ci.yml#L63 which checks the local html files. We either need to make the local build have the same folder structure or maybe figure out a baseurl to pass to the verify-links script.

**Comment by heaths** [comment]
I opted to just keep `baseurls` the same as it would be in the public site i.e. "azure-sdk" so I'll go ahead and update the script parameter.

File: releases/2021-01/dotnet.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ If you have a bug or feature request for one of the libraries, please [file an i
 - Added support for `AzureSasCredential`. That allows SAS rotation for long living clients.
 
 
-### Live Video Analytics on IoT Edge [Changelog](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/mediaservices/Azure.Media.Analytics.Edge/CHANGELOG.md#100-beta1-2020-12-11)
```
Comment (weshaggard) [inline releases/2021-01/dotnet.md:113]: I fixed these broken links earlier https://github.com/Azure/azure-sdk/commit/3d5dd5f21ed2dd10974963702cfde80894b8967b.

Comment (heaths) [reply releases/2021-01/dotnet.md:113]: My CSV fix actually includes the correct link. Any reason not to take my fixes? The other file will just end up being an effective noop.

File: _data/releases/latest/dotnet-packages.csv [hunk 3]
```diff
@@ -32,7 +32,7 @@
 "Azure.Security.KeyVault.Certificates","4.1.1","4.2.0-beta.6","Key Vault - Certificates","Key Vault","keyvault","","","client","true","","01/09/2020","active","","Microsoft.Azure.KeyVault",""
 "Azure.Security.KeyVault.Keys","4.1.1","4.2.0-beta.6","Key Vault - Keys","Key Vault","keyvault","","","client","true","","10/30/2019","active","","Microsoft.Azure.KeyVault",""
 "Azure.Security.KeyVault.Secrets","4.1.1","4.2.0-beta.5","Key Vault - Secrets","Key Vault","keyvault","","","client","true","","10/30/2019","active","","Microsoft.Azure.KeyVault",""
-"Azure.Media.Analytics.Edge","","1.0.0-beta.1","Media Analytics Edge","Media","mediaservices","","","client","true","","","","","",""
+"Azure.Media.Analytics.Edge","","1.0.0-beta.1","Media Analytics Edge","Media","https://github.com/Azure/azure-sdk-for-net/tree/Azure.Media.Analytics.Edge_1.0.0-beta.1/sdk/mediaservices/Azure.Media.Analytics.Edge","","","client","true","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:35]: Is this one needed?

File: _data/releases/latest/dotnet-packages.csv [hunk 4]
```diff
@@ -291,7 +291,7 @@
...
 "Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring","3.1.0","3.2.0-preview.4","LUIS - Authoring","Language Understanding","https://github.com/Azure/azure-sdk-for-net/tree/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring_3.2.0-preview.3/sdk/cognitiveservices/Language.LUIS.Authoring","NA","","client","false","","08/30/2019","active","","",""
 "Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime","","3.1.0-preview.1","LUIS Runtime","Language Understanding","https://github.com/Azure/azure-sdk-for-net/tree/Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime_3.1.0-preview.1/sdk/cognitiveservices/Language.LUIS.Runtime","NA","","client","False","","","","","",""
-"Microsoft.Azure.Media.LiveVideoAnalytics.Edge","","1.0.4-preview.1","Media Live Video Analytics Edge","Media Services","NA","NA","","client","False","","","","","",""
+"Microsoft.Azure.Media.LiveVideoAnalytics.Edge","","1.0.4-preview.1","Media Live Video Analytics Edge","Media Services","https://github.com/Azure/azure-sdk-for-net/tree/Microsoft.Azure.Media.LiveVideoAnalytics.Edge_1.0.4-preview.1/sdk/mediaservices/Microsoft.Azure.Media.LiveVideoAnalytics.Edge","NA","","client","False","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:294]: I'm fine taking this fix as it is a little better then mine which essentially removed the link.

**Review by weshaggard — APPROVED** [review]
Only one question about the track 2 link but either way it looks like that has moved directories so I will have to fix it up after its next release either way so I'm fine with what you have currently.

File: _data/releases/latest/dotnet-packages.csv (hunk 3 above)
Comment (heaths) [reply _data/releases/latest/dotnet-packages.csv:35]: Yes - it's also no longer in master.

Comment (weshaggard) [reply _data/releases/latest/dotnet-packages.csv:35]: Looks like it is still in master but just moved https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge

**Pull request merged**